Jobs/MM_job.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
+
+#   // INITIAL DESCRIPTION //
+"""Run Singlepoint Calculations"""
+
+#   // MEATDATA //
+__author__ = 'Alina Jansen'
+__date__ = '2024-09-19'
+
+#   // IMPORTS //
+
+#   Imports Of Existing Libraries
+import re
+import os
+import math
+import sqlite3
+import sys
+import subprocess
+import numpy as np
+
+#   Imports From Existing Libraries
+
+#   Imports Of Custom Libraries
+
+#   Imports From Custom Libraries
```

Comment:
In case you would want to consider `isort` to sort imports automatically, these comments could definitely go.